### PR TITLE
View polygon ind

### PR DESCRIPTION
**Pull request by aappling-usgs**
i think this will be everything we need to deal with the OS differences in `sf::st_transform`

### CODE REVIEW

**Comment by aappling-usgs** [comment]
I'm intentionally not commiting new or changed build/status files because i'd like @dblodgett-usgs or @jread-usgs to rebuild the spatial stuff (mainly for the sake of the precip data) on one of their Macs

File: 1_fetch.yml [hunk 1]
```diff
@@ -29,51 +29,58 @@ targets:
...
 
   view_config:
     command: viz_config[I(c('bbox', 'projection', 'width', 'height'))]
 
+  1_fetch/out/view_polygon.rds.ind:
+    command: post_view_polygon(target_name, view_config)
+  1_fetch/out/view_polygon.rds:
+    command: gd_get('1_fetch/out/view_polygon.rds.ind')
+  view_polygon:
+    command: get_view_polygon('1_fetch/out/view_polygon.rds.ind')
```
Comment (lindsaycarr) [inline 1_fetch.yml]: I know that you needed to make secondary_geoms and focus_geoms have files and indicator files in order to get around the OS issues with rebuilding, but does that mean we can't do the third step here for those and get them back as an object to pass around?

Comment (aappling-usgs) [reply 1_fetch.yml]: i think we could. but is it worth it for secondary_geoms and focus_geoms? those get used a lot less than view_polygon. that's why i didn't do it for all three.

Comment (aappling-usgs) [reply 1_fetch.yml]: taking it to the extreme, every file in this whole project could have that third step. but then everybody would always need to download every file, reducing some of the benefit of the shared cache. so i think drawing the line closer to very few cases like `view_polygon` is a good idea.

**Pull request merged**